### ISSUE TITLE
Totalterminal in fullscreen 

### ISSUE THREAD

**Issue by juanpabloaj** [binaryage/totalterminal#24]
How I can use Total terminal in full screen ? 

why don't add a hotkey to this?

**Comment by chrischambers** [comment]
+1 - I'd love to be able to toggle from visor-view to full screen and back. Visor view is great for quickly running commands, but it'd be nice to 'zoom in' and make use of all my screen real-estate for heavy terminal sessions.

**Comment by mikaa123** [comment]
+1
This would be much appreciated!

**Comment by funkotron** [comment]
+1 This is standard in the linux `guake` and is a must-have!

**Comment by n00shie** [comment]
Thanks @darwin!

**Comment by funkotron** [comment]
Thanks @darwin :)

**Comment by sydanny** [comment]
Ugh, it was working splendidly (Mavericks) and then it got stuck. CMD+OPT+F was working as a toggle - full screen, back to half, full, etc. But now I'm stuck at full screen and it won't shrink again. Once this process is done running, I'll quit it and restart. :(

**Comment by phx** [comment]
The same thing happened to me in Mavericks.  I've used this app for a long time in Snow Leopard and Mountain Lion with no issues, but in Mavericks, I toggled fullscreen, and it got stuck.  Killing the program didn't fix it for me either.  I even uninstalled and reinstalled 3 times with reboots in between, and I'm still stuck on full screen.  I'm a a network engineer and developer, so this is kind of embarrassing if there's something I'm overlooking here.  I just want my "bottom-stretch" back!
